src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -68,7 +68,8 @@ azure-mgmt-recoveryservicesbackup==5.1.0
 azure-mgmt-redhatopenshift==1.2.0
 azure-mgmt-redis==14.1.0
 azure-mgmt-relay==0.1.0
-azure-mgmt-resource==22.0.0
+azure-mgmt-reservations==2.0.0
```

Comment:
May I ask why do you need to add this SDK package `azure-mgmt-reservations`?

Reply:
This was unintended, I went ahead and removed reservations.